apps/aecore/src/aec_peer_connection.erl

```diff
@@ -436,7 +436,7 @@ handle_request(S, {Request, Args}, From) ->
     case get_request(S, Request) of
         none ->
             handle_request(S, Request, Args, From);
-        {MappedRequest, _From} ->
+        {MappedRequest, _From, _TRef} ->
```

Comment:
I see [setter](https://github.com/aeternity/epoch/blob/9736a29a264819e9474539e09f4905f057f03ae5/apps/aecore/src/aec_peer_connection.erl#L302) and other usages. Ok.

Any chance of having state with type specs?
(BTW Also I currently prefer record over map for process state - as more structured - though debatable.)